Sources/App/Models/Todo.swift

```diff
@@ -1,26 +1,19 @@
-import FluentSQLite
+import Fluent
 import Vapor
 
-/// A single entry of a Todo list.
-final class Todo: SQLiteModel {
-    /// The unique identifier for this `Todo`.
-    var id: Int?
+final class Todo: Model, Content {
+    static let schema = "todos"
+    
+    @ID(key: .id)
+    var id: UUID?
```

Comment:
Why changing from `Int` id to `UUID`?

Reply:
Fluent's default is now UUID, it works better across all the supported databases